.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - vendor
+    - $HOME/.composer/cache
+
+matrix:
+    include:
+        - php: 5.5
+        - php: 5.6
+        - php: 7.0
+        - php: hhvm
+    allow_failures:
+        - php: hhvm
+    fast_finish: true
+
+env:
+    - TEST_DIR=ezpublish/yaml_to_content_type
+
+before_install:
+    - phpenv config-rm xdebug.ini
```

Comment:
@htollefsen I suppose it might improve performance somewhat. I don't mind having that option in Travis CI, just make sure that it does not get executed for hhvm, as that option causes some problems there.

Something like this should do the trick (although haven't tested):

```
before_install:
    - if [ `phpenv version-name` != "hhvm" ]; then phpenv config-rm xdebug.ini; fi;
```